lints/lint_issuer_dn_leading_whitespace.go

```diff
@@ -0,0 +1,85 @@
+// lint_issuer_dn_leading_whitespace.go
+
+package lints
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+	"strings"
+)
+
+type IssuerDNLeadingSpace struct {
+	// Internal data here
+}
+
+func (l *IssuerDNLeadingSpace) Initialize() error {
+	return nil
+}
+
+func (l *IssuerDNLeadingSpace) CheckApplies(c *x509.Certificate) bool {
+	return true
+}
+
+func (l *IssuerDNLeadingSpace) RunTest(c *x509.Certificate) (ResultStruct, error) {
+	for _, country := range c.Issuer.Country {
+		if strings.HasPrefix(country, " ") {
+			return ResultStruct{Result: Warn}, nil
+		}
+	}
+	for _, org := range c.Issuer.Organization{
```

Comment:
Please run `go fmt` on this code? I'm starting to get frustrated at the number of times we've had to ask for this to be run in this project. Maybe there should be a test.